include/bluetooth/uuid.h

```diff
@@ -1781,6 +1862,78 @@ struct bt_uuid_128 {
  */
 #define BT_UUID_CCID \
 	BT_UUID_DECLARE_16(BT_UUID_CCID_VAL)
```

Comment:
Personally I don't think we need to change it. It may be independent of TBS, but it is placed in the same order as the full official list of UUIDs. 

Personally I would have defined it separately, instead of where it is now, but I think it is better to simply follow the SIG ordering.